dipy/workflows/denoise.py

```diff
@@ -103,13 +102,14 @@ def run(self, input_files, bvalues_files, bvectors_files, sigma=0,
             Default 0: it means sigma value estimation with the Manjon2013
             algorithm [3]_.
         b0_threshold : float, optional
-            Threshold used to find b=0 directions (default 0.0)
+            Threshold used to find b=0 directions
```

Comment:
Missing period. There are still a few missing.

Reply:
Addressed in e5af17f